From 5374b49c740ade918dc041c02bf25608e0b6b3ca Mon Sep 17 00:00:00 2001
From: Florian Sylvestre <fsylvestre@baylibre.com>
Date: Tue, 25 Apr 2023 15:04:52 +0200
Subject: [PATCH 03/10] goil: move inclusion of 'tpl_vector_table.s' in board
 specific config

Not all cortex-a platforms support secure mode, so the vector table is platform
dependent: then move its inclusion to board directory.
---
 goil/templates/config/cortex-a/armv7/config.oil | 1 +
 goil/templates/config/cortex-a/config.oil       | 1 -
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/goil/templates/config/cortex-a/armv7/config.oil b/goil/templates/config/cortex-a/armv7/config.oil
index 0a65dd33..c3613111 100755
--- a/goil/templates/config/cortex-a/armv7/config.oil
+++ b/goil/templates/config/cortex-a/armv7/config.oil
@@ -3,5 +3,6 @@
 CPU armv7 {
   PLATFORM_FILES armv7 {
     PATH = "cortex-a/armv7";
+    CFILE = "tpl_vector_table.s";
   };
 };
diff --git a/goil/templates/config/cortex-a/config.oil b/goil/templates/config/cortex-a/config.oil
index f6c118c9..a4e25f67 100755
--- a/goil/templates/config/cortex-a/config.oil
+++ b/goil/templates/config/cortex-a/config.oil
@@ -38,7 +38,6 @@ CPU arm_interrupt {
     CFILE = "tpl_system_call.S";
     CFILE = "tpl_stacks.S";
     CFILE = "tpl_trusted_fct.S";
-    CFILE = "tpl_vector_table.s";
     CFILE = "tpl_machine_arm.c";
   };
 
-- 
2.34.1

